src/renderers/WebGLRenderer.js

```diff
@@ -2719,8 +2719,9 @@ function WebGLRenderer( parameters ) {
 
 	this.copyFramebufferToTexture = function ( position, texture, level ) {
 
-		var width = texture.image.width;
-		var height = texture.image.height;
+		var levelScale = Math.pow( 2, - level );
```

Comment:
`level` should be an optional value. Right now, the [official example](https://threejs.org/examples/webgl_framebuffer_texture) breaks because `Math.pow( 2, - level );` produces an `NaN` value. Please write the method like so:

```js
this.copyFramebufferToTexture = function ( position, texture, level ) {

    level = level || 0;

    var levelScale = Math.pow( 2, - level );
    var width = texture.image.width * levelScale;
    var height = texture.image.height * levelScale;
    var glFormat = utils.convert( texture.format );

    textures.setTexture2D( texture, 0 );

    _gl.copyTexImage2D( _gl.TEXTURE_2D, level, glFormat, position.x, position.y, width, height, 0 );

};
```